transaction/interfaces.py

```diff
@@ -176,21 +183,17 @@ def note(text):
         appended to `.description`.
         """
 
-    def setUser(user_name, path="/"):
-        """Set the user name.
-
-        path should be provided if needed to further qualify the
-        identified user.  This is a convenience method used by Zope.
-        It sets the .user attribute to str(path) + " " + str(user_name).
-        This sets the `.user` attribute; see its docs for more detail.
-        """
-
```

Comment:
The changelog doesn't call out dropping `setUser` from the interface:  is that change intended?